### PR TITLE
Cf6

### PR DESCRIPTION
**Pull request by tomcojocar**

### CODE REVIEW

File: Kismet_CF_6.tex [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+% begin personnel required section -- {{{
+\subsection{Personnel Required}
+The test operations team consists of nine personnel:
+\begin{checklist}
+    \item The \opsfull{} directs operations procedures and communicates with the other test personnel.
+    \item The \primaryfull{} is the main system operator. \primary{} operates all manual valves as well as the test control system.
+    \item The \secondaryfull{} is the backup for \primary{}, and communicates with OPS. If \primary{} becomes incapacitated, \secondary{} is responsible for removing them from danger.
+    \item The \daqfull{} monitors and operates the test data acquisition system.
+    \item the \controlfull{} operates the test control system, including actuation of remote valves.
```
Comment (jacobdeery) [inline Kismet_CF_6.tex]: "the" -> "The"

File: Kismet_CF_6.tex [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    \end{checklist}
+    \item \ops: Give the system control key to \control{}.
+    \item \control{}: Engage the key switch and power on the control boxes.
+    \item \control{}: Open the Motorized Vent Valve.
+    \item \control{}: Open the Remote Fill Valve.
+    \begin{checklist}[label=$\bullet$]
+        \item If leaks are observed:
+        \begin{checklist}
+            \item \control{}: Close the Remote Fill Valve.
+            \item \primary{}: Open the Injector Valve.
```
Comment (jacobdeery) [inline Kismet_CF_6.tex]: I like to make this "Open the Injector Valve using the ropes" just to make it clear that they shouldn't need to walk up to the system

File: Kismet_CF_6.tex [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    \item \ops{}: Proceed only when a white plume is visible from the vent plug.
+    \item \control{}: Close the Motorized Vent Valve.
+    \item \control{}: Close the Remote Fill Valve.
+    \item \heat: Open the Tank Heating Valve.
+    \item \daq{}: Proceed only when the oxidizer tank pressure is at least 750 psi.
+    \begin{checklist}[label=$\bullet$]
+        \item If the oxidizer tank pressure does not reach 750 psi:
+        \begin{checklist}
+            \item \heat: Close the Tank Heating Valve.
+            \item \primary{}: Open the Injector Valve.
```
Comment (jacobdeery) [inline Kismet_CF_6.tex]: Same as above

File: Kismet_CF_6.tex [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+            \item \primary{} and \secondary: Approach the test plumbing.
+            \item \primary{}: Close the Cylinder Valve.
+            \item \control{}: Open the Remote Fill Valve.
+            \item \daq{}: Confirm that the Oxidizer Tank Pressure Transducer is reading atmospheric pressure.
+            \item \ops{}: Abort test procedures and revisit plumbing setup.
+        \end{checklist}
+    \end{checklist}
+    \item \heat: Close the Tank Heating Valve.
+    \item \textbf{\textit{PAUSE POINT}}
+    \item \primary{}: Conduct the cold flow test by opening the Injector Valve.
```
Comment (jacobdeery) [inline Kismet_CF_6.tex]: Same as above

File: Kismet_CF_6.tex [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+    \item \control{}: Open the Motorized Vent Valve.
+    \item \ops{}: Proceed only when the oxidizer tank has fully vented.
+    \item \primary{} and \secondary: Approach the test plumbing.
+    \item \primary{}: Close the Cylinder Valve.
+    \item \primary{}: Slowly open the Parallel Fill Valve.
+    \item \control{}: Open the Remote Fill Valve.
+    \item \primary{}: Disconnect the fill line from the supply cylinder.
+    \item \primary{}: Replace the cap on the carbon dioxide supply cylinder.
+    \item \daq{}: Confirm that the Fill Pressure Transducer is reading atmospheric pressure.
+    \item \ops{}: Wait for at least 3 minutes before proceeding.
```
Comment (jacobdeery) [inline Kismet_CF_6.tex]: we can remove this

File: Kismet_CF_6.tex [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+    \item Ensure that \ops{} is in possession of the system control key.
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end prior to start section -- }}}
+
+\newpage
+% begin Cold Flow Test Procedure section -- {{{
+\subsection{Cold Flow Test Procedure}
+\begin{checklist}
+    \item \ops{}: Confirm that the camera equipment is set up.
```
Comment (jacobdeery) [inline Kismet_CF_6.tex]: also, move this to right after the pause point (item 30). We shouldn't start recording until the last possible minute, to ensure we maximize battery life

File: Kismet_CF_6.tex (hunk 5 above)
Comment (tomcojocar) [reply Kismet_CF_6.tex]: @jacobdeery Line 194?

**Pull request merged**